version_control/katalyst_by_codurance/css/modules/services-cards.css

```
@@ -1,117 +1,111 @@
-        gap: var(--space-m);
-    }
-
-    .col-3 {
-        grid-column: span 3;
-    }
 
-    .col-4 {
-        grid-column: span 4;
-    }
-
-    .service-cards__card {
-        align-self: start;
-        transition: scale .2s,
-                    outline .2s;
-    }
-{% endcall %}    
-
-
-
-.service-cards__card:hover,
-.service-cards__card:focus {
-    outline: 2px solid var(--orange);
-    scale: 1.05;
-    transition: scale .3s,
-                outline .2s; 
+/* Positioning of card inner graphics */
+.service-cards__card:nth-child(1) img {
+    left: -6%;
+    top: -32%;
 }
 
-.icon-wrapper {
-    position: relative;
+.service-cards__card:nth-child(4) img {
+    left: 18%;
+    rotate: 322deg;
+    top: -33%;
 }
 
-.service-cards__card:hover h3,
-.service-cards__card:focus h3 {
-    margin-top: var(--space-l);
-    transition: margin .2s ease-in-out;
+.service-cards__card:nth-child(6) img {
+    rotate: 317deg;
+    right: -6%;
+    top: 77%;
 }
 
-.triangle {
-    position: absolute;
-    opacity: 0; 
-    top: var(--space-xs);
-    left: calc(-1 * var(--space-s));
-    z-index: -1;
-    animation: fade-out-backwards .2s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
+.service-cards__card:nth-child(7) img {
+    rotate: 50deg;
+    right: -6%;
+    top: -17%;
 }
 
-.service-cards__card:hover .triangle,
-.service-cards__card:focus .triangle {
-    opacity: 1;
-    animation: fade-in-forward .4s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
-}
+{% call breakpoints.small_and_medium() %}
+    .service-cards__card:nth-child(1) img {
+        left: -11%;
+        top: -40%;
+    }
+
+    .service-cards__card:nth-child(4) img {
+        top: -50%;
+    }
+
+    .service-cards__card:nth-child(6) img {
+        right: -12%;
+        top: 85%;
+    }
+
+    .service-cards__card:nth-child(7) img {
+        right: -24%;
+        top: -26%;
+    }
+{% endcall %}
+/* End positioning of card inner graphics */
 
-.icon-wrapper i {
-    font-size: var(--fs-3);
-    color: var(--orange);
+.service-cards__card i {
+    font-size: var(--fs-4);
     position: relative;
 }
 
 .services-cards h3 {
     font-family: {{theme.global_fonts.primary.font}};
-    font-weight: var(--fw-semibold);
+    font-weight: var(--fw-medium);
     font-size: var(--fs-1);
     margin-top: var(--space-s);
     margin-bottom: 0;
-    transition: margin .2s ease-in-out;
+    position: relative;
 }
 
 .services-cards p {
     margin-top: var(--space-xs);
+    position: relative;
 }
 
+{% call breakpoints.small_and_medium() %}
+    .services-cards article:not(:first-child) {
+        margin-top: var(--space-l);
+    }
+{% endcall %}
 
+{% call breakpoints.large_and_extra_large() %}
+    .services-cards {
+        display: grid;
+        grid-template-columns: repeat(12, 1fr);
+        gap: var(--space-l);
+    }
 
-@keyframes fade-in-forward {
-    0% {
-        opacity: 0;
-        transform: translateZ(-1400px);
+    .service-cards__card {
+        align-self: start;
     }
```

This rule is at the moment not applying or making any effect on the layout. Removing it it will set it to stretch by default in case we end up with longer text on the cards.
```suggestion
    .service-cards__card {
        align-self: stretch;
    }
```